arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,536 @@
+        "tags": {
+          "type": "object",
+          "additionalProperties": { "type": "string" },
+          "description": "Key value pairs of additional properties that can ebe specified"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RpPropertyBag",
+          "description": "Provision request properties"
+        }
+      }
+    },
+    "RpProvisionResult": {
+      "description": "Provision result specification",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Identifier for the operation"
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the operation"
+        },
+        "resourceType": {
+          "type": "string",
+          "description": "Type of the resource for Analysis Services"
+        },
+        "location": {
+          "type": "string",
+          "description": "Region in which the server operation was executed"
+        },
+        "sku": {
+          "$ref": "#/definitions/RpSkuTier",
+          "description": "The SKU name and tier"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "description": "Key-value pairs"
+          },
+          "description": "Custom tags specified as Key-value pairs"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RpPropertyBag",
+          "description": "Property bag for server operations"
+        }
+      }
+    },
+    "ProvisionResultList": {
+      "description": "Provision request specification",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/RpProvisionResult" },
+          "description": "The collection of returned provision operation results"
+        }
+      }
+    },
+    "ExceptionData": {
+      "description": "Details of an error in the operation",
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Error code"
+        },
+        "message": {
+          "type": "string",
+          "description": "Detailed message describing the error"
+        }
+      }
+    },
+    "OperationDetail": {
+      "description": "Details of the async operation specified by the operation id",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Identifier for the operation"
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the operation"
+        },
+        "status": {
+          "type": "string",
+          "description": "HTTP status code for the operation"
+        },
+        "startTime": {
+          "type": "string",
+          "description": "Start time of the operation"
```

Comment:
"startTime": { 
   "type": "string", 
   "format" : "date-time"
} 

This will indicate AutoRest to make startTime of a DateTime type in that language. It wont be a string type anymore.